### PR TITLE
framework && simple connection

### PR DESCRIPTION
**Pull request by trovwu**

### CODE REVIEW

**Review by trovwu — COMMENTED** [review]
init

File: src/main/java/io/milvus/param/R.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    public T getData() {
+        return data;
+    }
+
+    public void setData(T data) {
+        this.data = data;
+    }
+
+    public static <T> R<T> failed(){
+        R<T> r = new R<>();
```
Comment (xiaofan-luan) [inline src/main/java/io/milvus/param/R.java:35]: If failed, should we pass in the exception msg here?

**Pull request merged**